benchmark/spmv_comparison_cuda/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if (BUILD_CUDA)
+    add_executable(spmv_comparison_cuda spmv_comparison_cuda.cpp)
+    target_link_libraries(spmv_comparison_cuda ginkgo ${CUDA_RUNTIME_LIBS}
+        ${CUBLAS} ${CUSPARSE})
+    target_include_directories(spmv_comparison_cuda PRIVATE ${PROJECT_SOURCE_DIR})
```

Comment:
Seeing as the build failed with the CI system, you also probably need here either to propagate and use `CUDA_INCLUDE_DIRS` or to use Ginkgo's includes.